bin/run

```diff
@@ -1,14 +1,24 @@
-#!/usr/bin/env node
+#!/bin/sh
 
-const fs = require('fs');
-const path = require('path');
-const project = path.join(__dirname, '../tsconfig.json');
-const dev = fs.existsSync(project);
+# Get the local node_modules path;
+# if NODE_PATH is already set, append it to the existing value.
+if [ -n "$NODE_PATH" ]; then
+  UPDATED_PATH="${NODE_PATH}:${PWD}/node_modules"
```

Comment:
What is the ":" colon doing in the path?

Reply:
Appending the new value to the existing list of values. It works the same as `$PATH`. 
Something like this:
```
/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/lib/nodejs/node-v12.14.1-linux-x64/bin:/source/node_modules/.bin
```